testsuite/model/test-all-profiles.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 cd "$(dirname $0)"
+mvn -version
```

Comment:
Is this useful here or was added just for debug purposes?

Reply:
yes, this is intentional, since some of the failures might be only related to a given JDK / Maven version